api/v4/openapi.yaml

```diff
@@ -164,6 +227,10 @@ components:
           type: object
           additionalProperties:
             $ref: '#/components/schemas/Source'
+        unscanned:
```

Comment:
I think this should belong to the `Source`. Sources are the ones that provide the vulnerabilities. Besides it seems reasonable to also add `unscanned` to the `SourceSummary`

Reply:
Then, in case a provider has multiple `Source`, the `unscanned` list will be duplicated to each `Source` of the provider, which is fine, right?

Reply:
correct, we recommend the same package in each case